{"uid":"2a53f2928a42bf2b","name":"test_00_check_mine","fullName":"tests.test_01_mine.TestMine#test_00_check_mine","historyId":"a1081407c73a59b33579800ffeb41530","time":{"start":1656059294640,"stop":1656059298268,"duration":3628},"description":"检查我的页面元素","descriptionHtml":"<p>检查我的页面元素</p>\n","status":"broken","statusMessage":"Exception: click_my_tab() missing 1 required positional argument: 'self'","statusTrace":"args = (), kwargs = {}\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.i('--> %s' % func.__qualname__)\n>           return func(*args, **kwargs)\nE           TypeError: click_my_tab() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:83: TypeError\n\nDuring handling of the above exception, another exception occurred:\n\nargs = (<test_01_mine.TestMine object at 0x7ff44b673cd0>,), kwargs = {}\ngif_name = '/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'\nf = <_io.BufferedReader name='/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'>\nfile = b'GIF89ae\\x04\\x84\\t\\x87\\x00\\x00\\xff\\xff\\xff\\xfa\\xff\\xff\\xfd\\xfa\\xfe\\xfe\\xfe\\xfa\\xfe\\xfa\\xfa\\xfa\\xfa\\xfa\\xfe\\xfd\\xee\\xf...\\xd6\\xaa\\xad\\xde*\\xae\\xe6\\xaa\\xae\\xee*\\xaf\\xf6\\xaa\\xaf\\xfe*\\xb0\\x06\\xab\\xb0\\x0e+\\xb1\\x16\\xab\\xb1\\x1e+\\xb1\\x06\\x04\\x00;'\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.d('\\n--> %s', func.__qualname__)\n>           ret = func(*args, **kwargs)\n\ncommon/decorator.py:104: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test_01_mine.TestMine object at 0x7ff44b673cd0>\n\n    @allure.description(\"检查我的页面元素\")\n    @case\n    def test_00_check_mine(self):\n>       mine.check_mine_page()\n\ntests/test_01_mine.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def check_mine_page():\n>       init.click_my_tab()\n\npageObject/mine.py:162: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nargs = (), kwargs = {}\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.i('--> %s' % func.__qualname__)\n            return func(*args, **kwargs)\n        except requests.ConnectionError:\n            print(\"ConnectionError\")\n            reset_driver()\n            func(*args, **kwargs)\n        except Exception as e:\n            log.e('\\t<-- %s, %s, %s', func.__qualname__, 'Exception', e)\n>           raise Exception(e)\nE           Exception: click_my_tab() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:90: Exception\n\nDuring handling of the above exception, another exception occurred:\n\nargs = (<test_01_mine.TestMine object at 0x7ff44b673cd0>,), kwargs = {}\ngif_name = '/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'\nf = <_io.BufferedReader name='/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'>\nfile = b'GIF89ae\\x04\\x84\\t\\x87\\x00\\x00\\xff\\xff\\xff\\xfa\\xff\\xff\\xfd\\xfa\\xfe\\xfe\\xfe\\xfa\\xfe\\xfa\\xfa\\xfa\\xfa\\xfa\\xfe\\xfd\\xee\\xf...\\xd6\\xaa\\xad\\xde*\\xae\\xe6\\xaa\\xae\\xee*\\xaf\\xf6\\xaa\\xaf\\xfe*\\xb0\\x06\\xab\\xb0\\x0e+\\xb1\\x16\\xab\\xb1\\x1e+\\xb1\\x06\\x04\\x00;'\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.d('\\n--> %s', func.__qualname__)\n            ret = func(*args, **kwargs)\n            log.d('<-- %s, %s', func.__qualname__, 'Success')\n            return ret\n        except wda.WDAElementNotFoundError:\n            print(\"WDAElementNotFoundError\")\n            log.d('\\n--> %s', func.__qualname__)\n            ret = func(*args, **kwargs)\n            log.d('<-- %s, %s', func.__qualname__, 'Success')\n            return ret\n        except Exception as e:\n            log.e('<-- %s, %s, %s\\n', func.__qualname__, 'Exception', e)\n>           raise Exception(e)\nE           Exception: click_my_tab() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:115: Exception","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"xunit_setup_class_fixture_TestMine","time":{"start":1656059291995,"stop":1656059294639,"duration":2644},"status":"passed","steps":[],"attachments":[{"uid":"b9aa4911bc72419f","name":"init.check_login_status-20220624162813.PNG","source":"b9aa4911bc72419f.png","type":"image/png","size":2050274}],"parameters":[],"stepsCount":0,"attachmentsCount":1,"shouldDisplayMessage":false,"hasContent":true}],"testStage":{"description":"检查我的页面元素","status":"broken","statusMessage":"Exception: click_my_tab() missing 1 required positional argument: 'self'","statusTrace":"args = (), kwargs = {}\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.i('--> %s' % func.__qualname__)\n>           return func(*args, **kwargs)\nE           TypeError: click_my_tab() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:83: TypeError\n\nDuring handling of the above exception, another exception occurred:\n\nargs = (<test_01_mine.TestMine object at 0x7ff44b673cd0>,), kwargs = {}\ngif_name = '/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'\nf = <_io.BufferedReader name='/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'>\nfile = b'GIF89ae\\x04\\x84\\t\\x87\\x00\\x00\\xff\\xff\\xff\\xfa\\xff\\xff\\xfd\\xfa\\xfe\\xfe\\xfe\\xfa\\xfe\\xfa\\xfa\\xfa\\xfa\\xfa\\xfe\\xfd\\xee\\xf...\\xd6\\xaa\\xad\\xde*\\xae\\xe6\\xaa\\xae\\xee*\\xaf\\xf6\\xaa\\xaf\\xfe*\\xb0\\x06\\xab\\xb0\\x0e+\\xb1\\x16\\xab\\xb1\\x1e+\\xb1\\x06\\x04\\x00;'\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.d('\\n--> %s', func.__qualname__)\n>           ret = func(*args, **kwargs)\n\ncommon/decorator.py:104: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test_01_mine.TestMine object at 0x7ff44b673cd0>\n\n    @allure.description(\"检查我的页面元素\")\n    @case\n    def test_00_check_mine(self):\n>       mine.check_mine_page()\n\ntests/test_01_mine.py:21: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\n    def check_mine_page():\n>       init.click_my_tab()\n\npageObject/mine.py:162: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nargs = (), kwargs = {}\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.i('--> %s' % func.__qualname__)\n            return func(*args, **kwargs)\n        except requests.ConnectionError:\n            print(\"ConnectionError\")\n            reset_driver()\n            func(*args, **kwargs)\n        except Exception as e:\n            log.e('\\t<-- %s, %s, %s', func.__qualname__, 'Exception', e)\n>           raise Exception(e)\nE           Exception: click_my_tab() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:90: Exception\n\nDuring handling of the above exception, another exception occurred:\n\nargs = (<test_01_mine.TestMine object at 0x7ff44b673cd0>,), kwargs = {}\ngif_name = '/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'\nf = <_io.BufferedReader name='/Users/jike/PycharmProjects/pythonProject/TestReport/iPhoneX_cea307eb34fb7a0ebc2b6402e62de9bdb55cebfa/1656059296821.gif'>\nfile = b'GIF89ae\\x04\\x84\\t\\x87\\x00\\x00\\xff\\xff\\xff\\xfa\\xff\\xff\\xfd\\xfa\\xfe\\xfe\\xfe\\xfa\\xfe\\xfa\\xfa\\xfa\\xfa\\xfa\\xfe\\xfd\\xee\\xf...\\xd6\\xaa\\xad\\xde*\\xae\\xe6\\xaa\\xae\\xee*\\xaf\\xf6\\xaa\\xaf\\xfe*\\xb0\\x06\\xab\\xb0\\x0e+\\xb1\\x16\\xab\\xb1\\x1e+\\xb1\\x06\\x04\\x00;'\n\n    @wraps(func)\n    def wrapper(*args, **kwargs):\n        try:\n            log.d('\\n--> %s', func.__qualname__)\n            ret = func(*args, **kwargs)\n            log.d('<-- %s, %s', func.__qualname__, 'Success')\n            return ret\n        except wda.WDAElementNotFoundError:\n            print(\"WDAElementNotFoundError\")\n            log.d('\\n--> %s', func.__qualname__)\n            ret = func(*args, **kwargs)\n            log.d('<-- %s, %s', func.__qualname__, 'Success')\n            return ret\n        except Exception as e:\n            log.e('<-- %s, %s, %s\\n', func.__qualname__, 'Exception', e)\n>           raise Exception(e)\nE           Exception: click_my_tab() missing 1 required positional argument: 'self'\n\ncommon/decorator.py:115: Exception","steps":[],"attachments":[{"uid":"7ec2a38ac2fa179c","name":"init.click_my_tab-20220624162815.PNG","source":"7ec2a38ac2fa179c.png","type":"image/png","size":2050274},{"uid":"b214273504fc57d9","name":"case.gif","source":"b214273504fc57d9.gif","type":"image/gif","size":1947816},{"uid":"e016d79acc38514a","name":"log","source":"e016d79acc38514a.txt","type":"text/plain","size":509}],"parameters":[],"stepsCount":0,"attachmentsCount":3,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"xunit_setup_class_fixture_TestMine::0","time":{"start":1656059305871,"stop":1656059305871,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"tag","value":"jw"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_01_mine"},{"name":"subSuite","value":"TestMine"},{"name":"host","value":"jikedeMBP-4"},{"name":"thread","value":"32400-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_01_mine"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":["jw"]},"source":"2a53f2928a42bf2b.json","parameterValues":[]}